webui/src/Components/PNGInputField.tsx

```diff
@@ -47,16 +47,21 @@ export function PNGInputField({ min, max, onSelect, onError }: PNGInputFieldProp
 		return canvas.toDataURL()
 	}
 
-	const onClick = useCallback(() => {
-		onError(null)
-		if (inputRef.current) {
-			inputRef.current.click()
-		}
-	}, [onError])
+	const onClick = useCallback(
+		(e: React.MouseEvent) => {
+			onError(null)
+			if (inputRef.current) {
+				const form = e.currentTarget.getElementsByClassName('form-control')[0] as HTMLInputElement
```

Comment:
I think form is pointing to the same thing as `inputRef.current`, so it would be nicer to just use that.

Reply:
Oh. The search for css class in `onClick` is because there are two items in `e.currentTarget` while in `onClick` and that was the only way I could find to access the one I wanted. Either way works. I'll revert `onChange` now.